Makefile

```diff
@@ -142,6 +142,12 @@ test: manifests generate fmt vet envtest ## Run tests.
 test-e2e:
 	@export IMG_TAG=test-e2e; make docker-build; go test ./test/e2e/ -v -ginkgo.v --timeout 20m
 
+.PHONY: test-e2e-ocp
```

Comment:
Will this be a GPU or emulated e2e test?

Reply:
It is an emulated test

Reply:
Could we make the e2e argument a bit more clear on that end?